client/extensions/woocommerce/components/dashboard-widget/index.js

```diff
@@ -0,0 +1,146 @@
+/**
+ *
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Gridicon from 'gridicons';
+import { isUndefined, noop } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Popover from 'components/popover';
+import Tooltip from 'components/tooltip';
+
+class DashboardWidget extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			showDialog: false,
+			showTooltip: false,
+		};
+	}
+
+	hideTooltip = () => {
+		this.setState( { showTooltip: false } );
+	};
+
+	onSettingsPanelClose = () => {
+		this.setState( { showDialog: false } );
+		this.props.onSettingsClose();
+	};
+
+	setSettingsToggle = c => {
+		this.settingsToggle = c;
+	};
+
+	showTooltip = () => {
+		this.setState( { showTooltip: true } );
+	};
+
+	toggleSettingsPanel = () => {
+		const { showDialog } = this.state;
+		if ( showDialog ) {
+			this.props.onSettingsClose();
+		}
+		this.setState( { showDialog: ! showDialog } );
+	};
+
+	render() {
+		const {
+			children,
+			className,
+			image,
+			imageFlush,
+			imagePosition,
+			settingsPanel,
+			title,
+			translate,
+			width,
+		} = this.props;
+		const { showDialog, showTooltip } = this.state;
+		const isTopImage = image && 'top' === imagePosition;
```

Comment:
Minor: This is the only position that defines a variable and uses that to show the imageComponent. I think you should do the same thing with 'top` because it's a clear enough, and `isTopImage` isn't used anywhere else.